src/frontend/apps/impress/src/features/home/components/HomeSection.tsx

```diff
@@ -126,6 +126,7 @@ export const HomeSection = ({
             {title}
           </Text>
           <Text
+            as="div"
```

Comment:
I would recommend using a `p` tag instead of a `div` for better semantics.
If you want to keep the same visual style without the default margin applied to `p`, you can simply add:
```suggestion
           $css={css`
              margin: 0;
            `}
            as="p"
```

Reply:
I don't think we should use a "p" tag here, because it contains "div" and others "p" tag in it.

<img width="509" height="313" alt="image" src="https://github.com/user-attachments/assets/3cdbe3f9-a34c-4743-90ae-c3a22c140ba0" />